package ru.practicum.shareit.exception;

import org.junit.jupiter.api.Test;

public class ErrorRepsoneTest {
    @Test
    void isValidTest() {
    }
}
